builtin/logical/pki/path_fetch_issuers.go

```diff
@@ -94,10 +99,16 @@ their identifier and their name (if set).
 
 func pathGetIssuer(b *backend) *framework.Path {
 	pattern := "issuer/" + framework.GenericNameRegex(issuerRefParam) + "(/der|/pem|/json)?"
-	return buildPathGetIssuer(b, pattern)
+
+	displayAttrs := &framework.DisplayAttributes{
+		OperationPrefix: operationPrefixPKI,
+		OperationSuffix: "issuer-der|issuer-pem|issuer-json",
```

Comment:
Yes, I like this! 

I'm wondering if we want to call one of these `pki-fetch-issuer-der...`? Want something to differentiate it from the auth'd endpoints below.

Reply:
I've updated this to `"issuer|issuer-der|issuer-pem|issuer-json"` to match all 4 expansions of the pattern. The currently generated list is:

| Path | OperationIDs |
|------|--------------|
| "/{pki_mount_path}/issuer/{issuer_ref}" |        pki-read-issuer |
| | pki-write-issuer |
| | pki-delete-issuer |
| "/{pki_mount_path}/issuer/{issuer_ref}/der" |        pki-read-issuer-der |
| | pki-write-issuer-der |
| | pki-delete-issuer-der |
| "/{pki_mount_path}/issuer/{issuer_ref}/pem" |        pki-read-issuer-pem |
| | pki-write-issuer-pem |
| | pki-delete-issuer-pem |
| "/{pki_mount_path}/issuer/{issuer_ref}/json" |        pki-read-issuer-json |
| | pki-write-issuer-json |
| | pki-delete-issuer-json |

The `fetch` verb would not really apply to the write/delete cases. Which auth'd endpoints did you have in mind?